### DISCUSSION TITLE
MONDO_0019523 Walker-Warburg syndrome

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#1223] [General]
cc @maglott 

I think MONDO_0019523 Walker-Warburg syndrome is equivalent to [OMIM:236670 MUSCULAR DYSTROPHY-DYSTROGLYCANOPATHY (CONGENITAL WITH BRAIN AND EYE ANOMALIES), TYPE A, 1; MDDGA1](https://www.omim.org/entry/236670) and we should add the other synonyms from OMIM

**Comment by nicolevasilevsky** [comment]
@maglott 

I think MONDO_0000171 muscular dystrophy-dystroglycanopathy, type A and MONDO_0019523 | Walker-Warburg syndrome

are distinct classes though

**Comment by nicolevasilevsky** [comment]
see [spreadsheet](https://docs.google.com/spreadsheets/d/1s-jaBDNbJmQtv38uPrXOy7yiKZxbYJ940mfZeUt-T8U/edit#gid=510128657)

**Comment by kanems** [comment]
@nicolevasilevsky Following up on this... can we check with the OMIM team to get some clarification on whether WWS and MDDGA type are equivalent terms? Looking at the existing hierarchy/tree of children to MDDG type A MONDO:0000171, they all seem to be subtypes that match to the same Children of WWS MONDO:0019523.

**Comment by nicolevasilevsky** [comment]
@ahamosh could you please give us some advice on this? Thank you!

**Comment by ahamosh** [comment]
MDDGAs are WWS.  The number after MDDGA is which gene.  The MDDGBs are congenital muscular dystrophies with or without ID.  The MDDGCs are later-onset limb-girdle muscular dystrophies.  We lumped WWS and Muscle-Eye-Brain because the outcome difference largely depends upon medical intervention or lack thereof...

**Comment by nicolevasilevsky** [comment]
Thanks @ahamosh!

@kanems yes, the MDDG type A terms are children of both MDDG type A MONDO:0000171 and WWS MONDO:0019523.

@ahamosh or @kanems, are we missing a relationship between WWS and MDDG? Should MDDG type A be a subclass of WWS MONDO:0019523?

Thanks!

**Comment by ahamosh** [comment]
So I tried to look at this as graphs and trees and now I am confused.   As far as I can tell MONDO:0019523 is a generic WWS, which would be a generics MDDGA in OMIM, except that we don't have generics.  MDDGA1-13 are each of the WWS by gene.  We discuss the genetic heterogeneity in MIM:236670.  We also have them organized in a phenotypic series.  If this still isn't clear it might require a brief conversation where you explain the hierarchies and I can try to figure this out.

**Comment by nicolevasilevsky** [comment]
this is confusing! I drew a picture

![image](https://user-images.githubusercontent.com/6722114/100954777-1842c800-34ca-11eb-8986-65cbfbac3e12.png)

**Comment by ahamosh** [comment]
Yes!
Ada

**Comment by nicolevasilevsky** [comment]
@galyea-gard - FYI I merged MONDO:0019523 WWS  and MONDO:0000171 MDDG type A

**Comment by galyea-gard** [comment]
Hi Nicole!  I apologize that I did not see your message before.  We will still keep it in GARD because it is still used in the medical literature, but I will add it to the list for terms to review. While I believe you are right merging the terms MONDO:0019523 WWS and MONDO:0000171 MDDG type A I think that OMIM 236670 is not equivalent but a subtype of MDDG type A.  As commented, WWS can be caused by several different genes (hence the number after A, indicates the specific gene).  Please see phenotypic table: https://www.omim.org/phenotypicSeries/PS236670 
Thank you! Gioconda

**Comment by nicolevasilevsky** [comment]
Yes, we have OMIM 236670 represented as a subtype of MDDG type A (it is equivalent to MONDO_0009364 'muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A1').

Thanks for the feedback @galyea-gard!

**Comment by ahamosh** [comment]
Hi Nicole,
I think that the PS236670 (the whole series) is the overall matching of the overall WWA and MDDGA.  MDDGA1 is WWS1 or MDDGA1, which are equivalent.

**Comment by nicolevasilevsky** [comment]
Yes, it's now like this:
![image](https://user-images.githubusercontent.com/6722114/101567388-91886200-3985-11eb-9e83-9abfb5d7d383.png)

**Comment by ahamosh** [comment]
Perfect!
